Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx

```diff
@@ -0,0 +1,252 @@
+
+// -------------------- Styles -------------------- //
+
+const detailListContainer = css`
+  width: 50%;
+  height: 400px;
+  overflow: hidden;
+  float: left;
+  flex-grow: 1;
+`;
+
+const templateDetailContainer = css`
+  width: 50%;
+  height: 400px;
+  overflow: auto;
+  flex-grow: 1;
+  float: left;
+`;
+
+const pickerContainer = css`
+  position: relative;
+  height: 400px;
+  border: 1px solid #f3f2f1;
+`;
+
+const rowDetails = (disabled) => {
+  return {
+    root: {
+      color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+      selectors: {
+        '&:hover': {
+          background: disabled ? NeutralColors.white : NeutralColors.gray30,
+          color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+        },
+        '&.ms-DetailsRow.is-selected': {
+          background: disabled ? NeutralColors.white : NeutralColors.gray30,
+          color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+        },
+      },
+    },
+  };
+};
+
+const rowTitle = (disabled) => {
+  return {
+    cellTitle: {
+      color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+      selectors: {
+        ':hover': {
+          background: disabled ? NeutralColors.white : NeutralColors.gray30,
+          color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+        },
+      },
+    },
+  };
+};
+
+const tableCell = css`
+  outline: none;
+  :focus {
+    outline: rgb(102, 102, 102) solid 1px;
+  }
+`;
+
+const content = css`
+  outline: none;
+`;
+
+const optionKeys = {
+  createFromScratch: 'createFromScratch',
+  createFromQnA: 'createFromQnA',
+  createFromTemplate: 'createFromTemplate',
+};
+
+// -------------------- CreateOptions -------------------- //
+type CreateOptionsProps = {
+  templates: BotTemplate[];
+  onDismiss: () => void;
+  onNext: (data: string) => void;
+  fetchTemplates: (feedUrls?: string[]) => Promise<void>;
+  fetchReadMe: (moduleName: string) => Promise<void>;
+  templateReadMe: string;
+} & RouteComponentProps<{}>;
+
+export function CreateOptionsV2(props: CreateOptionsProps) {
+  const [option] = useState(optionKeys.createFromTemplate);
+  const [disabled] = useState(false);
+  const { templates, onDismiss, onNext, fetchReadMe, templateReadMe } = props;
+  const [currentTemplate, setCurrentTemplate] = useState('');
+  const [emptyBotKey, setEmptyBotKey] = useState('');
+  const [selectedFeed, setSelectedFeed] = useState<{ props: IPivotItemProps } | undefined>(undefined);
+
+  const selection = useMemo(() => {
+    return new Selection({
+      onSelectionChanged: () => {
+        const t = selection.getSelection()[0] as BotTemplate;
+        if (t) {
+          setCurrentTemplate(t.id);
+        }
+      },
+    });
+  }, []);
+
+  const handleJumpToNext = () => {
+    let routeToTemplate = emptyBotKey;
+    if (option === optionKeys.createFromTemplate) {
+      routeToTemplate = currentTemplate;
+    }
+
+    if (option === optionKeys.createFromQnA) {
```

Comment:
We need to figure out how this feature is surfaced.  This is what allows a user to start in the QNA maker mode.

Reply:
This should appear in the template list even though it is not in the feed.

Reply:
Ive created task #5506 to track this. I say we leave it for now and revisit it once we have an empty bot template which presumably will be used under the hood when this template is selected (instantiate empy bot template -> Route to import from qna modal and apply the uploaded KB to empty bot)